dev/TreeView/TestUI/TreeViewPage.xaml.cs

```diff
@@ -998,5 +998,20 @@ private void ExpandRootNode_Click(object sender, RoutedEventArgs e)
         {
             TestTreeViewItemsSource[0].IsExpanded = !TestTreeViewItemsSource[0].IsExpanded;
         }
+
+        private bool isTreeView1SourceInUse = true;
+        private void SwapItemsSource_Click(object sender, RoutedEventArgs e)
+        {
+            if (!isTreeView1SourceInUse)
```

Comment:
You can maybe drop the boolean field and use `if (this.ContentModeTestTreeView.ItemsSource == TestTreeView2ItemsSource)` instead?

Reply:
Updated :)